### PR TITLE
Use consistent naming for to/destination in 1559

### PR DESCRIPTION
**Pull request by axic**
Also noticed that the RLP section uses camelcase, while the python spec uses snakecase.

Furthermore other fields are also not matching, like `value` vs `amount`.

Basically asking for guidance, before I do more changes.

### CODE REVIEW

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - eip-1559.md requires approval from one of (vbuterin, econoar, afdudley, mslipper, i-norden, abdelhamidbakhta)
 @vbuterin @econoar @afdudley @mslipper @i-norden @abdelhamidbakhta

**Comment by MicahZoltu** [comment]
I think the inconsistency is a side effect of copying material from sources found elsewhere.  I like `destination` over `to` personally, though I'm sure the additional letters will bother some people.  Regarding snake vs camel, as I have mentioned before I think internal consistency (within a given EIP) is more important than cross-EIP consistency so I would support either all snake or all camel within this document.

As for whether to use camel or snake, I'm a bit more ambivalent.

**Review by youtacrsbyvars — APPROVED** [review]

**Review by youtacrsbyvars — APPROVED** [review]

**Review by youtacrsbyvars — APPROVED** [review]

**Review by youtacrsbyvars — APPROVED** [review]

**Comment by axic** [comment]
Should anything happen to this change?

Should I submit another PR for the camelcase question?

Or should I just close this?

**Review by alita-moore — APPROVED** [review]

**Comment by alita-moore** [comment]
oops thought this was something else, my approval doesn't mean much 😓

**Comment by MicahZoltu** [comment]
Sadly, despite EIP-1559 having half a dozen authors, it seems that none of them actually pay attention to GitHub notifications.  I have found that to get changes to 1559 merged you have to ping @timbeiko in Discord who then pesters one of the authors until they merge it.  Not an ideal situation, but in my experience that is the reality.

**Comment by timbeiko** [comment]
@abdelhamidbakhta can you review this?

**Review by abdelhamidbakhta — APPROVED** [review]
Looks good to me.

**Comment by abdelhamidbakhta** [comment]
> @abdelhamidbakhta can you review this?

Done. Sorry for the delay

**Comment by axic** [comment]
Thanks @abdelhamidbakhta! Should I open a separate one for value vs amount, and another for camelcase vs snakecase discrepancies?

**Comment by abdelhamidbakhta** [comment]
> Thanks @abdelhamidbakhta! Should I open a separate one for value vs amount, and another for camelcase vs snakecase discrepancies?

You can do those 2 changes in a single PR IMO.

**Pull request merged**
